v3/src/models/data/formula-utils.ts

```diff
@@ -180,15 +182,16 @@ export const getExtremeCollectionDependency =
 export const getFormulaChildMostAggregateCollectionIndex = (formulaCanonical: string, dataSet: IDataSet) => {
   const attrId = getExtremeCollectionDependency(formulaCanonical, dataSet, { order: "max", aggregate: true })
   const collectionId = dataSet.getCollectionForAttribute(attrId || "")?.id
-  return dataSet.getCollectionIndex(collectionId || "") ?? null
+  const collectionIndex = dataSet.getCollectionIndex(collectionId || "")
+  return collectionIndex !== -1 ? dataSet.getCollectionIndex(collectionId || "") : null
```

Comment:
Personal preference, but I generally use `>= 0` in situations like this because it reads more like _is a valid index_ than _isn't the magic sentinel value used to indicate invalid indices, which means it's valid_.

Reply:
My brain reads `-1` as an invalid index (kinda like checking against null or undefined), but I get your point. I'll update it soon.

Reply:
Fixed in https://github.com/concord-consortium/codap/commit/e71c56ac75dc1583af2252a09d1cb76f9f902f26 that will be part of another PR (minimizing chances of merge conflicts).